CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+# Contributing to plonk
+
+All of the code under this repository is licensed under the
+Mozilla Public License Version 2.0.
+
+If you have questions or comments, please feel free to email the
+authors listed [here](https://github.com/dusk-network/Poseidon252/blob/master/Cargo.toml#L4). 
+
+For feature requests, suggestions, and bug reports, please open an issue on
+[our Github](https://github.com/dusk-network/Poseidon252).  (Or, send us
+an email if you're opposed to using Github for whatever reason.)
+
+Patches are welcomed as pull requests on
+[our Github](https://github.com/dusk-network/Poseidon252), as well as by
+email (preferably sent to all of the authors listed in `Cargo.toml`).
+
+If you want to work on an issue, please let us know in the comments of it and
+we will assign it to you.
+
+If you're new to this repository and you want to contribute, you can
+look for the issues tagged the with `good first issue` label. We try to add this tag
+to the issues we belive are easy for newcomers.
+
+If you need any help regarding an issue feel free to ask the authors in the `Draft pull request`
+or the issue (or by email if you prefer it by any reason.)
+
+# Pull Requests
+
+- Keep your pull requests as `Draft` until they are ready to be reviewed.
+
+- Be descriptive in the titles of your PRs and also provide good descriptions about
```

Comment:
I would also add somewhere a reference to [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/)